mtg/store.go

```diff
@@ -320,6 +345,29 @@ func (s *SQLite3Store) UpdateTxWithOutputs(ctx context.Context, t *Transaction,
 		}
 	}
 
+	cmt := decimal.RequireFromString(change)
+	if cmt.Sign() > 0 {
+		c, err := s.countUnreceivedOutputs(ctx, tx)
+		if err != nil {
+			return err
+		}
+		out := &UnifiedOutput{
+			OutputId:             UniqueId(t.TraceId, "change"),
+			TransactionRequestId: t.TraceId,
+			TransactionHash:      t.Hash.String(),
+			OutputIndex:          1,
+			AssetId:              t.AssetId,
+			Amount:               cmt,
+			State:                SafeUtxoStateUnreceived,
+			Sequence:             uint64(c + 1),
```

Comment:
this sequence seems to be useless? And counting is incorrect because we delete old unreceived outputs. So we could just use some random or timestamp as sequence and avoid counting.